janitor/functions/conditional_join.py

```diff
@@ -279,6 +281,8 @@ def conditional_join(
             only appears in the right DataFrame, and `both` if the observation’s
             merge key is found in both DataFrames.
         force: If `True`, force the non-equi join conditions to execute before the equi join.
+        return_indices: Returns only the matching indices for the left and right dataframes.
```

Comment:
Should this argument be renamed to `return_matching_indices`, or `matching_indices`?

Reply:
I'll make the change to `return_matching_indices`